README.md

```diff
@@ -29,7 +29,6 @@ services:
       - BENCH_MODE=mixed
       - DURATION=5m
       - EXTRA_ARGS=--analyze.out=out/warp-s3-kube.csv
-      - KEEP_ALIVE_AFTER_TEST=false
```

Comment:
Why remove this var ? 😄

Reply:
Answered [here](https://github.com/qjoly/warp-s3-kube/pull/4#issuecomment-2610821465)